### PR TITLE
[Task] Update site header name theme common SA 5765 

### PR DESCRIPTION
**Pull request by RSAK56**
# Description
Modified site name
Fixes # (issue)
UI/UX
# Type of change
Enhancement
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] UI/UX modification
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: theme_override/mkdocs.yml [hunk 1]
```diff
@@ -1,4 +1,3 @@
-site_name: Multi Tenant Operator
```
Comment (ufateh) [inline theme_override/mkdocs.yml:1]: we wanted to keep empty name here. ''

**Review by karl-johan-grahn — APPROVED** [review]

File: theme_override/mkdocs.yml (hunk 1 above)
Comment (karl-johan-grahn) [reply theme_override/mkdocs.yml:1]: why?

Comment (RSAK56) [reply theme_override/mkdocs.yml:1]: It's an override for each site name since it's going to be different in for each in their header next to the logo

Comment (RSAK56) [reply theme_override/mkdocs.yml:1]: Added back

**Pull request closed without merging**